scripts/validate-licenses.ts

```diff
@@ -42,6 +42,7 @@ const licensesWhitelist = [
   // Have a full copyright grant. Validated by opensource team.
   'Unlicense',
   'CC0-1.0',
+  '0BSD',
```

Comment:
we might need to consult OSPO about this. I believe it's fine to use it, but it would be great to get a confirmation from @juliaferraioli or someone else from that part of the world.

I'll shoot an email to them now.